_schema.graphql

```diff
@@ -3354,15 +3354,23 @@ type FairCounts {
   ): FormattedNumber
 }
 
+type FairExhibitor {
+  # Exhibitor name
+  exhibitor_name: String
+
+  # Exhibitors id
+  exhibitor_id: String
+
+  # Partner default profile id
+  profile_id: String
+}
+
 type FairExhibitorsGroup {
   # Letter exhibitors group belongs to
   letter: String
 
-  # Exhibitors sorted by name
-  exhibitors: [String]
-
-  # Partner/Exhibitor default profile id
-  profile_ids: [String]
+  # The exhibitor data.
+  exhibitors: [FairExhibitor]
```

Comment:
This is going to be a breaking schema change, which _should_ be okay as long as we aren't consuming this part of the schema yet. Make sure to check Volt, Reaction, and Force before merging.

Reply:
It's not being used anywhere else.